BryantUniversity/BryantUniversity/Repo/CourseSectionRepo.cs

```diff
@@ -15,14 +15,15 @@ public CourseSectionRepo(Context context)
             _context = context;
         }
 
-        public CourseSection GetCourseSectionById (int courseSectionId)
+        public CourseSection GetCourseSectionById (int courseSectionId, int userId)
```

Comment:
You should rename this function because we're probably going to need a function that just gets a coursesection by its ID